src/legacy/ui/public/new_platform/set_services.test.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { __reset__, __setup__, __start__, PluginsSetup, PluginsStart } from './new_platform';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import * as dataServices from '../../../../plugins/data/public/services';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import * as visualizationsServices from '../../../../plugins/visualizations/public/services';
+import { LegacyCoreSetup, LegacyCoreStart } from '../../../../core/public';
+import { coreMock } from '../../../../core/public/mocks';
+import { npSetup, npStart } from './__mocks__';
+
+describe('ui/new_platform', () => {
```

Comment:
This suite will fail if any `getX` getter is exported from `plugins/visualizations/public/services` (or `plugins/data/public/services`) without also being added to `ui/new_platform/set_services`. 

This will prevent us from adding regressions to the legacy world if we update these services in the future.